cns/NetworkContainerContract.go

```diff
@@ -477,9 +477,10 @@ type PodIpInfo struct {
 }
 
 type HostIPInfo struct {
-	Gateway   string
-	PrimaryIP string
-	Subnet    string
+	Gateway     string
+	PrimaryIP   string
+	SecondaryIP string
```

Comment:
As we discussed, we do not need this secondaryIP right?

Reply:
yes i can remove it, was just waiting for your review & if other fields are good enough